src/cascadia/TerminalApp/SnippetsPaneContent.h

```diff
@@ -77,13 +77,14 @@ namespace winrt::TerminalApp::implementation
             }
         }
 
-        void UpdateFilter(const winrt::hstring& filter)
+        void UpdateFilter(std::shared_ptr<fzf::matcher::Pattern> pattern)
         {
-            _filteredCommand->UpdateFilter(filter);
+            _pattern = std::move(pattern);

```

Comment:
I'm sorry about that. If we move from `pattern` here we need to change all further references in this function to use `_pattern`. The original becomes untouchable/undefined. :)